wiki/Official_Songs/en.md

```diff
@@ -1,53 +1,66 @@
-|-------------------------------------------------------------------------------------------|
-| cYsmix - Classic Pursuit                                                                  |
-| Source: osu!                                                                              |
-| [Download .osz](http://puu.sh/kZrR3/3a45e0ec85.osz)                                       |
-| Bounty: bundled with osu! client, 50$ store credit, 6 months supporter tag, profile badge |
-
-| ![cYsmix BG 2](cYsmix_bg_2.jpg "cYsmix BG 2")                                             |
-|-------------------------------------------------------------------------------------------|
-| cYsmix - Fright March                                                                     |
-| Source: osu!                                                                              |
-| [Download .osz](http://puu.sh/kZrTX/13431bd8e6.osz)                                       |
-| Bounty: bundled with osu! client, 50$ store credit, 6 months supporter tag, profile badge |
-
-| ![cYsmix BG 2](cYsmix_bg_2.jpg "cYsmix BG 2")                                             |
-|-------------------------------------------------------------------------------------------|
-| cYsmix - Moonlight Sonata                                                                 |
-| Source: osu!                                                                              |
-| [Download .osz](http://puu.sh/kZrVh/d549b2e56f.osz)                                       |
-| Bounty: bundled with osu! client, 50$ store credit, 6 months supporter tag, profile badge |
-
-Bounties
-========
-
-**Bounties** are being offered to help create more original content for osu!. Songs listed with a bounty have been provided by the artists with full permission to be used and distributed, and are now waiting to be mapped by the community. Once a beatmap of one of these songs is ranked, it will be checked by osu!staff, and should the quality be deemed sufficient, the bounty will be awarded to the mapper. Beatmap Bounties are only offered to one mapper per mapset - collaborations do not count. The entire mapset must be mapped by *one mapper* for *one game mode* (either osu!, Taiko, Catch the Beat or osu!mania). Only one mapset per mode will receive the bounty per song. The background image of these songs will be chosen by the staff. The winner of each song will be determined by the staff exclusively.
-
-For more information, **[read the original blog post about it](http://blog.ppy.sh/post/129936251068/mapping-with-rewards-sep-2015)**
+# Official Songs
+
+**Official Songs** are songs officially approved by the artist to be used within *osu!* Every member of the community is allowed to pick up the songs and create their own beatmaps of them. Most of the songs are created exclusively for *osu!*, while some of them even have a bounty placed to encourage the mapping community creating the beatmaps for this songs.
+
+## Available Songs
```

Comment:
🤔 move the completed songs (the ones with a winner) to a new section called (`## Claimed Songs`, or similar)?